truss/templates/shared/logging.py

```diff
@@ -73,6 +114,6 @@ def setup_logging() -> None:
             logger.handlers.clear()
             logger.addHandler(JSON_LOG_HANDLER)
 
-        # some special handling for request logging
-        if logger.name == "uvicorn.access":
-            logger.addFilter(HealthCheckFilter())
+    # clear uvicorn loggers so we can overwrite
```

Comment:
Could you explain what the uvicorn.error logging handler was doing? Is that important?

Reply:
Yep. It just logs the `Application startup complete.` log.